Header header
std_msgs/Int32 id
std_msgs/String type  # Vehicle/Pedistrain/Bicycle...
State state
Bbox3D bbox_3d
#Predicted behavior and trajectory
std_msgs/Int32 lane_id  #If exists
